src/Compilers/Core/Portable/Diagnostic/DiagnosticSeverity.cs

```diff
@@ -27,7 +27,12 @@ public enum DiagnosticSeverity
         /// <summary>
         /// Something not allowed by the rules of the language or other authority.
         /// </summary>
-        Error = 3
+        Error = 3,
+
+        /// <summary>
+        /// Something that can be potentially improved.
+        /// </summary>
+        Suggestion = 4,
```

Comment:
Agreed - Unnecessary is orthogonal here, my understanding was incorrect.

However, can you comment on:
>
1. Do Info and Suggestion truly represent 2 different severities? Neither of them break my build and are identical in all ways except the representation in the editor.
2. As analyzer author, why should I have to care about info versus suggestion if all I want to do is report an issue that doesn't break end user's build?